integration_test/uninstall_test.go

```diff
@@ -47,6 +47,23 @@ func TestKrewUninstall(t *testing.T) {
 	}
 }
 
+func TestKrewUninstallPluginsFromCustomIndex(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test.WithEnv(constants.EnableMultiIndexSwitch, 1).WithIndex()
+	test.Krew("index", "add", "foo", test.TempDir().Path("index/"+constants.DefaultIndexName)).RunOrFail()
+	test.Krew("install", "foo/"+validPlugin).RunOrFail()
+
```

Comment:
I think you don't need this part of the test.

you can just run `uninstall a/b` without `b` being installed and do the

```
	if !strings.Contains(string(b), "INDEX/PLUGIN") {
		t.Error("expected warning about using canonical name to be in output")
	}

```

check you did in `TestKrewUpgradePluginsFromCustomIndex`